### PR TITLE
Simulation Analysis Scripts

### PR DESCRIPTION
**Pull request by gwaygenomics**
Scripts to facilitate running the simulation analysis for all data combinations

### CODE REVIEW

File: run_simulation.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-d', '--data', help='location of data file')
+parser.add_argument('-o', '--noise', help='amount of noise injection')
+parser.add_argument('-n', '--sample_size', help='number of samples')
+parser.add_argument('-g', '--num_genes', help='number of genes')
+parser.add_argument('-f', '--out_file', help='where to save output results')
+parser.add_argument('-s', '--seeds', default=5,
+                    help='number of different seeds to run on current data')
+parser.add_argument('-t', '--transform', default='zeroone',
```
Comment (danich1) [inline run_simulation.py]: Is this always a 'zeroone' transformation? If no what are the other options?

File: run_simulation.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+                    help='how many epochs for VAE models')
+parser.add_argument('-ae', '--adage_epochs', default=100,
+                    help='how many epochs for ADAGE models')
+parser.add_argument('-b', '--batch_size', default=25,
+                    help='How many samples to use to update weights')
+parser.add_argument('-r', '--vae_learning_rate', default=0.001,
+                    help='learning rate for the VAE models')
+parser.add_argument('-a', '--adage_learning_rate', default=1.2,
+                    help='learning Rate for the ADAGE models')
+parser.add_argument('-l', '--loss', default='mse',
```
Comment (danich1) [inline run_simulation.py]: Same as above.

File: run_simulation.py (hunk 1 above)
Comment (gwaygenomics) [reply run_simulation.py]: No, but it should _usually_ be. I will add [choices](https://docs.python.org/2/library/argparse.html#choices) to the parser though to make these clear!

**Comment by gwaygenomics** [comment]
Thanks @danich1 - I believe ac46c4e addresses your concerns

**Comment by gwaygenomics** [comment]
Thanks again @danich1 - I think I am going to go ahead and merge this as it is tightly related to and somewhat dependent on #112 

The actual data simulation and evals these scripts are facilitating are more important IMO

**Pull request merged**